po/pl.po

```diff
@@ -556,6 +571,15 @@ msgstr "Grupa audio %d"
 msgid "Audio group name"
 msgstr "Nazwa grupy audio"
 
+msgid "Audio recording:"
+msgstr "Nagrywanie audio"
+
+msgid "Auto add new devices"
+msgstr "Auto dodawać nowe urządzenia"
```

Comment:
Although this one works, I'd suggest changing "dodawać" to either "dodawanie" (in line with "zarządzanie" below) or "dodawaj" (imperative).

Reply:
```suggestion
msgstr "Auto dodawanie nowych urządzeń"
```

Reply:
I think all settings should keep a single style: either imperatives or the nouns. Please choose it and suggest your variants with the
```
+
-
```
icon

Reply:
Well, let me take a second then and I'll go through the entire file to align them.